API-Samples/C#/ArmBased/README.md

```diff
@@ -53,6 +53,68 @@ For more information visit [here](https://docs.microsoft.com/en-us/azure/media-s
 Outline the required components and tools that a user might need to have on their machine in order to run the sample. This can be anything from frameworks, SDKs, OS versions or IDE releases.
 -->
 
+# Authentication Model
+
+This sample demonstrates two methods for authenticating the code to the Video Indexer account.
+
+1. Using Default Azure Credentials: This method utilizes the logged-in user or a User Assigned Managed Identity/System Assigned Identity associated with the running host.
+2. Using Service Principal Authentication (Entra App Registration)
+
+## Authentication with Default Azure Credentials
+
+1. Ensure that you are logged in to your Azure subscription by running the `az login` command.
+2. If you are using the same user across multiple tenants, set the `tenantId` variable under the `AccountTokenProvider.cs` class.
+3. Extract the Object ID of the logged-in user or Managed Identity and proceed to Section 3 below.
+
+## Authentication with Service Principal
+
+1. Create an Azure Entra ID App that will be used as a service principal.
+
+```
+az ad sp create-for-rbac --role Owner --display-name $appName --scopes /subscriptions/$SUBSCRIPTION_ID
+```
+
+2. After the command finishes, write down the values for appId (clientId) and password that will be used in your code.
+
+
+![](entra_app.png){: width="300" height="300"}
+
+
+```
+servicePrincipalId=$(az ad sp list --display-name $appName --query "[0].id" -o tsv)
```

Comment:
Is this to get the objectId? I think you need to add a line explaining it

Reply:
fixed.